solution/backend/regulations/templates/regulations/login.html

```diff
@@ -0,0 +1,33 @@
+{% extends "regulations/base.html" %}
+{% load static %}
+
+{% block spa_styles %}
+    <link href="https://cdn.jsdelivr.net/npm/@mdi/font@4.x/css/materialdesignicons.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="{% static '/vite/index.css' %}" />
+{% endblock %}
+
+{% block title %}Single sign-on to CMCS | Medicaid & CHIP eRegulations{% endblock %}
+
+{% block header %}
+    {% include "regulations/partials/header.html" %}
+{% endblock %}
+
+{% block body %}
+<div class="login-container">
+        <div class="nav-container">
+            <div class="content">
+                <h1>Sign in with your CMS account</h1>
+            </div>
+        </div>
+        <div>
+            <main class="login">
+                <form action="{% url 'oidc_authentication_init' %}" method="get">
+                    <input type="hidden" name="next" value="{{ next }}">
+                    <button type="submit" class="action-btn default-btn">Continue to CMS IDM</button>
+                </form>
+                <p>Requires an eRegulations job code, available to Policy Repository pilot participants.</p>
+                <p>For help, contact stephanie.boyd@cms.hhs.gov or julian.rodriguez@a1msolutions.com.</p>
+            </main>
+        </div>
+</div>
+{% endblock %}
```

Comment:
This red "do not enter" icon means that there is not a `newline` at the end of the file.  We all talked last year about ensuring that our text editors were set up to automatically add `newline`s to the end of new files. 

Here's how to enable this setting in Jetbrains IDEs: https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file